new-lamassu-admin/src/pages/Machines/Machines.js

```diff
@@ -38,32 +35,34 @@ const GET_INFO = gql`
         label
         type
       }
+      latestEvent {
+        note
+      }
     }
     config
   }
 `
 
-const getMachines = R.path(['machines'])
+const getMachineID = path => path.slice(path.lastIndexOf('/') + 1)
```

Comment:
When this happens, the page renders without any information and a GraphQL error is shown on the console, since the 'device_id' property is null thus the query can't be satisfied.